### ISSUE TITLE
Empty results with getMyFeesEstimates

### ISSUE THREAD

**Issue by nikgilbe** [jlevers/selling-partner-api#432]
I have a script that checks fees in batch mode that has been working successfully until "recently" (I'm not sure exactly when it stopped unfortunately). Having checked this morning, I have just noticed that all the responses back from SP-API are blank. If I check a fee individually, I get results OK. Has anyone else experienced anything similar? Not sure if this is an Amazon problem or something else...

Any thoughts greatly appreciated.

## Error:

```
This is the result I'm getting:
Array
(
    [0] => SellingPartnerApi\Model\FeesV0\FeesEstimateResult Object
        (
            [container:protected] => Array
                (
                    [status] =>
                    [fees_estimate_identifier] =>
                    [fees_estimate] =>
                    [error] =>
                )
        )
)
```

## Code

```php
This is the function I am using:
function checkBunkerFeeBatch($skuArray, $apiInstance) {
        for ($i = 0; $i < count($skuArray); $i++) {
                $batchArray[$i] =
                    [
                        'FeesEstimateRequest' => [
                            'MarketplaceId'       => 'A1F83G8C2ARO7P',
                            'IsAmazonFulfilled'   => false,
                            'PriceToEstimateFees' => new \SellingPartnerApi\Model\FeesV0\PriceToEstimateFees([
                                'listing_price' => new \SellingPartnerApi\Model\FeesV0\MoneyType([
                                    'currency_code' => 'GBP',
                                    'amount'        => 200,
                                ]),
                                'shipping'      => new \SellingPartnerApi\Model\FeesV0\MoneyType([
                                    'currency_code' => 'GBP',
                                    'amount'        => 0,
                                ]),
                            ]),
                            'Identifier'          => $skuArray[$i]['sku'],
                        ],
                        'IdType'              => "SellerSKU",
                        'IdValue'             => $skuArray[$i]['sku'],
                    ];
        }
        try {
                $result= $apiInstance->getMyFeesEstimates($batchArray);
                print_r($result);
        } catch (Exception $e) {
                echo 'Exception when calling FeesApi->getMyFeesEstimates: ', $e->getMessage(), PHP_EOL;
		}

and sample input to the functions is:
Array
(
[0] => Array
	(
	[FeesEstimateRequest] => Array
	(
		[MarketplaceId] => A1F83G8C2ARO7P
		[IsAmazonFulfilled] =>
		[PriceToEstimateFees] => SellingPartnerApi\Model\FeesV0\PriceToEstimateFees Object
	(
	[container:protected] => Array
	(
		[listing_price] => SellingPartnerApi\Model\FeesV0\MoneyType Object
		(
		[container:protected] => Array
		(
			[currency_code] => GBP
			[amount] => 200
		)
	)

		[shipping] => SellingPartnerApi\Model\FeesV0\MoneyType Object
		(
		[container:protected] => Array
		(
			[currency_code] => GBP
			[amount] => 0
		)
	)

		[points] =>
		)
	)
		[Identifier] => ABC123456XYZ
	)

	[IdType] => SellerSKU
	[IdValue] => ABC123456XYZ
	)
)```

## Seller Central SP API config page screenshot

*your screenshot here*

**Comment by nikgilbe** [comment]
This turned out to be the same issue / resolution as in the below one ....

https://github.com/jlevers/selling-partner-api/issues/297#issuecomment-1237560806

**Comment by jlevers** [comment]
This should be fixed in `v5.5.3`. Let me know if you continue to run into issues.
